### PR TITLE
Fix assignments

### PR DESCRIPTION
**Pull request by shirayu**
The assignment of text and label seems to be opposite.

### CODE REVIEW

**Comment by yoheikikuta** [comment]
Thanks for sending your PR.

I guess this issue is related to https://github.com/yoheikikuta/bert-japanese/issues/4 (sorry this issue is written in Japanese).
Which python version do you use?  Not greater than 3.6?

Regarding this issue, I'm now planning to explicitly fix the order of the Dataframe of finetuing data.

**Comment by shirayu** [comment]
Sorry, I missed the issue.
Yes, the Python version is `3.5.3`, and this is the very same issue to #4.

I changed the code to use header line to get indices.
All of `data/livedoor/dev.tsv`, `data/livedoor/test.tsv`,  and `data/livedoor/train.tsv` have a header line.
Is the comment `Only the test set has a header` wrong?

**Comment by yoheikikuta** [comment]
Thank you!
Yes, we have the header line in data for finetuning.
So how about reading indices of text/label within i==0 condition as below? This is a simpler solution.

```
def _create_examples(self, lines, set_type):
"""Creates examples for the training and dev sets."""
  examples = []
  for (i, line) in enumerate(lines):
    # Get indices of text/label from the header line.
    if i == 0:
      idx_text = line.index('text')
      idx_label = line.index('label')
    guid = "%s-%s" % (set_type, i)
    ...
```

> Is the comment Only the test set has a header wrong?

Exactly, thanks.

File: src/run_classifier.py [hunk 1]
```diff
@@ -237,12 +237,13 @@ def _create_examples(self, lines, set_type):
     """Creates examples for the training and dev sets."""
     examples = []
     for (i, line) in enumerate(lines):
-      # Only the test set has a header
       if i == 0:
+        idx_text = lines[0].index('text')
+        idx_label = lines[0].index('label')
         continue
```
Comment (yoheikikuta) [inline src/run_classifier.py]: Could you remove this `continue` line?

**Comment by yoheikikuta** [comment]
Thanks for your contribution!
